packages/angular/package.json

```diff
@@ -42,25 +42,17 @@
     "@kolkov/angular-editor": "^1.1.5",
     "@mat-datetimepicker/core": "^4.1.0",
     "@mat-datetimepicker/moment": "^4.1.0",
-    "@material-ui/codemod": "^4.5.0",
-    "@material-ui/core": "^4.11.0",
-    "@material-ui/icons": "^4.9.1",
-    "@material-ui/lab": "^4.0.0-alpha.56",
-    "@material-ui/styles": "^4.10.0",
-    "@material-ui/system": "^4.9.14",
-    "@material-ui/types": "^5.1.0",
-    "@material-ui/utils": "^4.10.2",
-    "@ngstack/code-editor": "^2.1.1",
+    "@ngstack/code-editor": "^1.0.0",
     "@stratusjs/boot": "^0.2.5",
     "@stratusjs/core": "^0.2.18",
-    "@stratusjs/map": "^0.3.0",
+    "@stratusjs/map": "^0.0.1",
```

Comment:
the map was reverted to 0.0.1 which would break a number of things if it wasn't manually called in multiple package.jsons

Reply:
I updated the map package in my last commit and what it was before was `0.0.1`, so I reverted that line to a couple days ago.  I'll update it again.  :smile: